src/Controllers/MagicLinkController.php

```diff
@@ -50,6 +50,8 @@ public function __construct()
      */
     public function loginView()
     {
+        $this->displayErrorMagicLinkDisabled();
```

Comment:
`RedirectResponse` must be returned from this method if `allowMagicLinkLogins` is false.